elixir/lib/mock_system.ex

```diff
@@ -0,0 +1,5 @@
+defmodule GithubRepoCloner.MockSystem do
+  def cmd("sh", ["-c", command]) do
+    {:ok, "Command executed: #{command}"}
```

Comment:
This just returns something I can assert against in a test. We don't need to actually run a system command in test.